app/src/main/java/net/bible/android/view/activity/readingplan/actionbar/ReadingPlanTitle.kt

```diff
@@ -31,12 +31,11 @@ import android.widget.TextView
 
 import net.bible.android.activity.R
```

Comment:
What plans do you have for this action bar? Planning to keep it like this for now?